keyboards/handwired/meck_tkl/meck_tkl.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 Gabriel Kim (@gabrielkim13)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "quantum.h"
+
+/* This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT( A0, A1, A2, A3, A4, A5, A6, A7, A8, A9, A10, A11, A12, A13, A14, A15, B0, B1, B2, B3, B4, B5, B6, B7, B8, B9, B10, B11, B12, B13, B14, B15, B16, C0, C1, C2, C3, C4, C5, C6, C7, C8, C9, C10, C11, C12, C13, C14, C15, C16, D0, D1, D2, D3, D4, D5, D6, D7, D8, D9, D10, D11, D12, E0, E1, E2, E3, E4, E5, E6, E7, E8, E9, E10, E11, E12, F0, F1, F2, F3, F4, F5, F6, F7, F8, F9, F10) \
```

Comment:
while not necessary, it's highly recommended to format this to match the keyboard's layout.

Reply:
Using the `kxy` naming convention would also be preferred; these might be interpreted as pin names by the preprocessor or other people.